robottelo/ui/login.py

```diff
@@ -45,7 +45,7 @@ def is_logged(self):
             for cookie in self.browser.get_cookies()
         )
         # construct the link to the Dashboard web page
-        url_root = settings.server.get_url() + '/dashboard'
+        url_root = settings.server.get_url() + '/about'
```

Comment:
In 6.2 we have some issues with dashboard, so point to another page. And dashboard is not default page anymore